packages/config/config/devices/0x0234/zdb5400.json

```diff
@@ -0,0 +1,581 @@
+{
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "ZDB5400",
+	"description": "Wall Controller",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0124"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Report, Pushbutton 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Binary Switch Report, Pushbutton 1",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Multi-Level Report, Pushbutton 1",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Basic Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Binary Switch Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"7": {
+			"label": "Multilevel Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"8": {
+			"label": "Basic Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"9": {
+			"label": "Binary Switch Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"10": {
+			"label": "Multilevel Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"11": {
+			"label": "Basic Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"12": {
+			"label": "Binary Switch Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"13": {
+			"label": "Multilevel Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"14": {
+			"label": "Dimmer Basic Report",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1[0xff]": {
+			"$import": "templates/logic_group_template.json#dimmer_pushbutton"
+		},
```

Comment:
No overlaps please

Reply:
This can be removed from the code. references overlapping bit masks